lua/browse/input.lua

```diff
@@ -18,6 +18,6 @@ M.search_input = utils.callback_search(function(input)
     end
 
     return path
-end)
+end, {})
```

Comment:
Can we also support this here? It is when someone calls `input_search` directly.

Reply:
According to <https://github.com/lalitmee/browse.nvim/pull/16/commits/abc8d0948b9e5ec047fa43dff743754b001dae0b>, now `input_search` can be called by user with visual mode support.
e.g.
```lua
    keys = {
      {
        mode = { "n", "x" },
        "<leader>I",
        function() require("browse").input_search() end,
      },
    },
```